tsconfig.json

```diff
@@ -1,16 +1,42 @@
 {
   "compilerOptions": {
-    "outFile": "./dist/types/index",
+    "alwaysStrict": false,
+    "declaration": true,
+    "declarationMap": true,
+    "importHelpers": true,
+    "inlineSources": true,
+    "isolatedModules": true,
+    "lib": [
+        "ES2022",
+        "dom"
+    ],
+    "moduleResolution": "node10",
```

Comment:
**Question**

Was setting a specific node runtime the fix here? (as opposed to an ES version/year?)

I struggled with this - there seems to be a few combinations between this config and the consumer that might work, but it seemed almost arbitrary.

Reply:
Did some experimenting and had trouble with `NodeNext`, but `Node16` worked. So that's what we'll go with for now.